### DISCUSSION TITLE
Onboarding PR not re-created

### DISCUSSION THREAD

**Discussion by sekhavati** [renovatebot/renovate#12993] [Request Help]
Hi there! 👋 

I'm suffering from the same issue discussed [here](https://github.com/renovatebot/renovate/discussions/11119). ie: I renamed the closed PR & delete the branch, but the onboarding PR has failed to re-trigger since (albeit its only been about 90 mins since I did it). 

At what time interval would the re-trigger happen? Or if possible, please could you manually invoke it? 

Repo details:
github.com 
< redacted >
private repo

Thanks in advance for any help.

**Comment by rarkins** [comment]
done

**Reply by sekhavati** [reply]
Thank you @rarkins

**Comment by HonkingGoose** [comment]
@sekhavati the edit history for the comment still shows the name of your private repository!

GitHub Docs explains how you can edit your comment history so that the name is no longer visible. [^gh-docs]
I recommend you read the docs and make the change.

The repository location/existence is not secret anymore as people (and bots) with `read` access will have seen it by now.

[^gh-docs]: https://docs.github.com/en/communities/moderating-comments-and-conversations/tracking-changes-in-a-comment#deleting-sensitive-information-from-a-comments-history

**Reply by sekhavati** [reply]
@HonkingGoose you make a good point and I didn't realise that feature existed in GitHub! thank you for making me aware of it. 

The redacted information is not overly sensitive but wanted to remove it for good measure
